zetaclient/config/types.go

```diff
@@ -55,6 +55,11 @@ type SolanaConfig struct {
 	Endpoint string `mask:"filled"`
 }
 
+// SUIConfig is the config for SUI chain
```

Comment:
```suggestion
// SuiConfig is the config for Sui chain
```
Sui is a word (did the same mistake originally) let's try to fix it wherever possible